projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/date-time-field/date-time-field.component.css

```diff
@@ -18,15 +18,65 @@
   border-radius: 24px;
 }
 
-.datetime-icon {
+/* .datetime-icon {
   font-size: 22px;
-}
+} */
 
-/* TODO: this is only a temporary fix for Angular datetime picker style issues. Remove this if there is better solution.*/
+/* TODO: this is only a temporary fix for Angular datetime picker style issues. Remove this if there is better solution. */
 ::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-body-cell {
```

Comment:
did you try removing this style, there's a chance they've fixed whatever issue that was causing this

Reply:
Yes, we can remove this. 
I also noticed that the button colors were not consistent b/w date-time and date, so i added those as well